payment-failure/src/main/scala/com/gu/identity/paymentfailure/LambdaService.scala

```diff
@@ -33,11 +33,20 @@ class LambdaService(sqsService: SqsService, sendEmailService: SendEmailService)
 
 object LambdaService {
 
-  def fromConfig(config: Config): LambdaService = {
+  def default(config: Config): LambdaService = {
     val identityClient = new IdentityClient(config)
     val sqsService = new SqsService(config)
     val brazeClient = new BrazeClient(config)
-    val sendEmailService = new SendEmailService(identityClient, brazeClient, config)
+    val sendEmailService = new DefaultBrazeEmailService(identityClient, brazeClient, config)
+    new LambdaService(sqsService, sendEmailService)
+  }
+
+  def encryptedEmailTest(config: Config): LambdaService = {
+    val identityClient = new IdentityClient(config)
+    val sqsService = new SqsService(config)
+    val brazeClient = new BrazeClient(config)
+    val encryptedEmailTest = new EncryptedEmailTest(identityClient)
+    val sendEmailService = new BrazeEmailServiceWithEncryptedBrazeEmailTest(brazeClient, encryptedEmailTest, config)
```

Comment:
In future, if we want to change the change the `BrazeEmailService`, just create a new implementation and inject it into the `LambdaService`.